test/presentation/core/core_fixtures.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:bloc_test/bloc_test.dart';
+import 'package:collaction_app/application/auth/auth_bloc.dart';
+import 'package:collaction_app/application/crowdaction/subscription/subscription_bloc.dart';
+import 'package:collaction_app/application/crowdaction/subscription_status/subscription_status_bloc.dart';
+
+class MockAuthBloc extends MockBloc<AuthEvent, AuthState> implements AuthBloc {}
```

Comment:
We could call this file `mocks.dart` since it just contains mocks not fixtures

Reply:
Agreed.

I saw some mocks in the test utilities file, they don't belong there.